app/models/conversion_host.rb

```diff
@@ -8,7 +8,7 @@ class ConversionHost < ApplicationRecord
 
   belongs_to :resource, :polymorphic => true
   has_many :service_template_transformation_plan_tasks, :dependent => :nullify
-  has_many :active_tasks, -> { where(:state => ['active', 'migrate']) },
+  has_many :active_tasks, -> { where(:state => ['active', 'migrate', 'pending']) },
```

Comment:
Maybe an `active_task` is a task that already has a conversion_host ?

```
has_many :active_tasks, -> { where.not(:conversion_host => nil) }
```

Reply:
@agrare that's true. And `active_tasks` is only used for throttling, so we could simply use `has_many :tasks` as you suggest.